auditlog/models/log.py

```diff
@@ -44,3 +49,19 @@ class AuditlogLogLine(models.Model):
     field_name = fields.Char(u"Technical name", related='field_id.name')
     field_description = fields.Char(
         u"Description", related='field_id.field_description')
+
+    @api.model
+    def create(self, vals):
+        """ Ensure field_id is not empty on creation. """
+        res = super(AuditlogLogLine, self).create(vals)
+        if not res.field_id or res.field_id is None:
```

Comment:
I think the first clause covers the second clause already.

Reply:
I always get confused when None is compared equally to False.

Reply:
`None` and `False` are both falsy but sometimes you need to distinguish `None` from `False` in which case you use `is None`